intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -167,7 +167,8 @@ \section{First Time connection to the \hpc}
       \end{center}
 \fi
 
-    \item \firsttimeconnection
+    \item If this is your first time connecting, you will be asked to verify the authenticity of
+        the login node. Please see ~\ref{sec:warning-message-new-host} on how to do this.
```

Comment:
`Please see section~\ref{...}`, otherwise you only get a section number here, no?